examples/index.md

```diff
@@ -126,3 +126,206 @@ Content-Type: application/vnd.api+json
 for readability. In practice, these characters must be percent-encoded, as
 noted in the base specification.
 
+## Error Objects
+
+Examples of how [error objects](http://jsonapi.org/format/#error-objects) work.
+
+### Responses to failed resource creation:
+
+Response with one error object:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title":  "Invalid Attribute",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    }
+  ]
+}
+```
+
+Notice that the attribute `first-name` is dasherized.  Dasherized attributes are preferred
+over underscored.
+
+Notice also that the detail is specific to this occurrence of the problem,
+whereas the title is more generic.
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "id": "ds.errors.invalid-error-expects-json-api-format",
```

Comment:
Oh, looks like I somehow mixed up code and id here and in the error codes section as well